server/forest/User.js

```diff
@@ -0,0 +1,12 @@
+const Liana = require('forest-express-sequelize');
+
+Liana.collection('User', {
```

Comment:
Can you explain what this will do?

Reply:
https://doc.forestadmin.com/developer-guide/lumber.html#what-is-a-smart-action